Add a blurp about hovered list actions (#405)
pxblue/doc-it: src/docs/patterns/lists.mdx

> If your entire list item is clickable, you should not include local list actions, as the two interactions can interfere with one another.

> On desktop applications, you may use hover events to display applicable list actions. Hover list actions can save spaces, leaving more room for displaying static information.

Reviewer: ```suggestion
On desktop applications, you may use hover events to display applicable list actions. Hover list actions can save space, leaving more room for displaying static information.
```

<ImageGrid
    images={[HoverActionsImage]}
    caption={

>         'Hover actions can save more room on a desktop application, and save your application from repeated columns of action buttons; but the nature of it prevents this pattern from being used over touch screen devices like mobile phones and tablets.'

Reviewer: ```suggestion
        'Hover actions can save room on desktop applications and allow you to avoid repeated columns of action buttons, but the nature of hover prevents this pattern from being used on touch screen devices.'
```
